File: api/src/main/java/org/pmiops/workbench/config/AppEngineConfig.java
package org.pmiops.workbench.config;

import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
import com.google.api.client.http.HttpTransport;
import java.io.IOException;
import java.security.GeneralSecurityException;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class AppEngineConfig {

  @Bean
  HttpTransport httpTransport() throws GeneralSecurityException, IOException {
    return GoogleNetHttpTransport.newTrustedTransport();
  }
}

File: api/src/main/java/org/pmiops/workbench/config/AppEngineConfig.kt
package org.pmiops.workbench.config

import com.google.api.client.extensions.appengine.http.UrlFetchTransport
import com.google.api.client.http.HttpTransport
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class AppEngineConfig {

    @Bean
    internal fun httpTransport(): HttpTransport {
        return UrlFetchTransport.getDefaultInstance()
    }
}
